docs/Developers/Rankings.rst

```diff
@@ -0,0 +1,103 @@
+Rankings
+========
+
+The coala community is split into different ranks which have different rights
+and obligations. This page will explain which the ranks are and what you need
+to do to get promoted to a higher rank.
+
+Currently the ranks can be seen `here <https://github.com/orgs/coala/teams>`__
+and are as it follows:
+
+ - newcomers
+ - developers
+ - maintainers
+ - admins
+
+The Newcomer Rank
+-----------------
+
+Everyone starts here. This is the rank where you can get a promotion easiest
+and which has the lowest obligations, and, obviously, the lowest rights.
+
+Rights
+~~~~~~
+
+The newcomers have no write access, and no direct labelling rights. The only
+thing that can be done by them is reviewing by leaving useful comments on
+Pull Requests which help them prove themselves as good reviewers.
+
+How to get promoted
+~~~~~~~~~~~~~~~~~~~
+
+If you complete the following tasks, you will become a developer at coala:
+
+  - run coala on a project of yours
+  - merge a ``difficulty/newcomer`` Pull Request
+  - review at least a ``difficulty/newcomer`` Pull Request
+  - merge a ``difficulty/low`` Pull Request
+  - review at least a ``difficulty/low`` or higher Pull Request
+
+When you ran coala on a project, please fill our
+`usability survey <http://coala.io/usability>`_. Once you got your first Pull
+Request merged successfully, fill in our
+`survey form <http://coala.io/newform>`_. With that you can help us making your
+experience better!
+
+Once you have achieved all these, just ask for being promoted on the chat and
+provide links to your reviews and merged Pull Requests. Then, you will be able
+to name yourself a coala developer!
+
+The Developer Rank
+------------------
+
+The developers team consists of people that have already proven themselves,
+both on the coding side and the reviewing side.
+
+Rights
+~~~~~~
+
+The developers can:
+
+ - use cobot to label PRs
+ - acknowledge commits that they are ready to be merged
+
+For more information about this, simply type ``cobot explain review``
+on https://coala.io/chat.
+
+How to get promoted
+~~~~~~~~~~~~~~~~~~~
+
+There is no simple process that you have to complete in order to become a
+maintainer, such as there is to become a developer. In order to get promoted
+to a maintainer, you need to:
+
+ - be a reliable community member
+ - review PRs consistently (not necessarily a PR each day, but a
+   few each week at least)
+ - merge at least a ``difficulty/medium`` Pull Request
+
+Once you've done these, at least two maintainers need to vouch for you so you
+are included in the maintainers team too.
+
+The Maintainer Rank
+-------------------
+
+The maintainers team is the team that coordinates mostly what code goes into
```

Comment:
building on @nemaniarjun 's comment, remove `is the team that` then move `mostly` in front of `coordinates`.